### PR TITLE
[UXE-3960] test: add error responses test

### PR DESCRIPTION
**Pull request by aloisio-m-bastian**
### What is the new behavior introduced by this PR?
Test edge application error responses page
### Does this PR introduce breaking changes?
- [x] No
- [ ] Yes 

### Does it have a link on Figma?

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (feat, test, refactor, etc)
- [ ] Application responsiveness was tested to different screen sizes
- [x] Code is formatted and linted
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

**Comment by aloisio-m-bastian** [comment]
Link da execução do test manual: https://github.com/aziontech/azion-console-kit/actions/runs/9931092584

File: cypress/e2e/edge-application.cy.js [hunk 1]
```diff
@@ -139,6 +139,53 @@ describe('Edge Application', { tags: ['run',] }, () => {
     cy.get(selectors.edgeApplication.searchInput).type(originName)
     cy.get(selectors.edgeApplication.tableRowAddress).should('have.text', 'test2.com')
   })
+  it.only('should add an error response', () => {
```
Comment (william-tome) [inline cypress/e2e/edge-application.cy.js:142]:
```suggestion
  it('should add an error response', () => {
```

File: cypress/e2e/edge-application.cy.js [hunk 2]
```diff
@@ -139,6 +139,53 @@ describe('Edge Application', { tags: ['run',] }, () => {
...
+
+    //act
+    //add error response 1
+    cy.get(selectors.edgeApplication.addErrorResponse).click()
+    cy.get(selectors.edgeApplication.errorResponseStatusCodes(1)).click()
+    cy.get(selectors.edgeApplication.errorResponseStatusCodes(1)).find('li').eq(0).click()
+    cy.get(selectors.edgeApplication.errorResponsePaths(1)).type('/test/')
+    cy.get(selectors.edgeApplication.errorResponseCustomStatus(1)).type('200')
+
+    //add erro response 2
```
Comment (william-tome) [inline cypress/e2e/edge-application.cy.js:171]:
```suggestion
    //add error response 2
```

File: cypress/e2e/edge-application.cy.js [hunk 3]
```diff
@@ -139,6 +139,53 @@ describe('Edge Application', { tags: ['run',] }, () => {
...
+    cy.get(selectors.edgeApplication.searchInput).type(edgeApplicationName)
+    cy.get(selectors.edgeApplication.tableRowName).should('have.text', edgeApplicationName)
+
+    //add error response
+    //arrange
+    cy.get(selectors.edgeApplication.tableRowLastEditor).click()
+    cy.get(selectors.edgeApplication.tabOption(3)).click()
+
+    //act
+    //add error response 1
```
Comment (william-tome) [inline cypress/e2e/edge-application.cy.js:164]: Seria bom nomear o error response ao invés de enumerar

**Review by william-tome — APPROVED** [review]

**Review by HerbertJulio — APPROVED** [review]

**Pull request merged**